project/AppDependencies.scala

```diff
@@ -4,38 +4,37 @@ import sbt._
 
 object AppDependencies {
 
-  val akkaVersion = "2.6.15"
+  val akkaVersion            = "2.6.19"
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-28" % "6.4.0",
-    "com.github.java-json-tools" % "json-schema-validator"     % "2.2.14",
-    "uk.gov.hmrc.mongo"          %% "hmrc-mongo-play-28"         % "0.68.0",
-    "uk.gov.hmrc"                %% "play-json-union-formatter"  % "1.15.0-play-28",
-    "com.typesafe.play" %% "play-json-joda" % "2.9.2",
-    "com.lightbend.akka" %% "akka-stream-alpakka-mongodb" % "3.0.1",
-    "com.typesafe.akka" %% "akka-slf4j" % akkaVersion,
-    "com.typesafe.akka" %% "akka-actor" % akkaVersion,
-    "com.typesafe.akka" %% "akka-stream" % akkaVersion,
-    "com.typesafe.akka" %% "akka-protobuf-v3" % akkaVersion,
-    "org.reactivemongo" %% "reactivemongo-akkastream" % "0.20.13",
-    "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.12.4",
-    "com.typesafe.akka"          %% "akka-actor-typed"              % akkaVersion,
-    "com.typesafe.akka"          %% "akka-serialization-jackson"              % akkaVersion
+    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28"   % "7.1.0",
```

Comment:
I am not 100% sure but I think bootstrap 7.x has some breaking changes or something so we upgraded all repos to 6.4 at this point

Reply:
From https://github.com/hmrc/bootstrap-play#version-700, it looks safe to use unless we're using crypto-play-xxx

Reply:
to me this changelog looks not really safe we might run in some encryption problems somewhere